hack/multikueue-e2e-test.sh

```diff
@@ -0,0 +1,239 @@
+function cluster_kueue_deploy {
+    kubectl config use-context kind-${1}
+    kubectl apply --server-side -k test/e2e/config
+}
+
+function kueue_deploy {
+    (cd config/components/manager && $KUSTOMIZE edit set image controller=$IMAGE_TAG)
+
+    cluster_kueue_deploy $LEADER_KIND_CLUSTER_NAME
+    cluster_kueue_deploy $WORKER1_KIND_CLUSTER_NAME
+    cluster_kueue_deploy $WORKER2_KIND_CLUSTER_NAME
+
+    #wait for the manager to start
+    # failing a this point might indicate a manifestation of
+    # https://kind.sigs.k8s.io/docs/user/known-issues/#pod-errors-due-to-too-many-open-files
+    kubectl config use-context kind-$LEADER_KIND_CLUSTER_NAME
+    kubectl wait -n kueue-system deployment kueue-controller-manager --for condition=Available --timeout=120s
+    kubectl config use-context kind-$WORKER1_KIND_CLUSTER_NAME
+    kubectl wait -n kueue-system deployment kueue-controller-manager --for condition=Available --timeout=120s
+    kubectl config use-context kind-$WORKER2_KIND_CLUSTER_NAME
+    kubectl wait -n kueue-system deployment kueue-controller-manager --for condition=Available --timeout=120s
+}
+
+trap cleanup EXIT
+startup
+kind_load
+kueue_deploy 
+
+#TODO: run some actual tests
+#$GINKGO --junit-report=junit.xml --output-dir=$ARTIFACTS -v ./test/multikueue-e2e/...
+
+
+#TODO: remove the following lines
+
+# even if waiting for deployment, the webhook server might not be up, wait an extra 10 sec
+sleep 10
+
+kubectl config use-context kind-$LEADER_KIND_CLUSTER_NAME
+kubectl run kubectl-to-1 --image registry.k8s.io/kubectl:v1.28.4 --overrides=\
+'
+{
+	"spec": {
+		"volumes": [
+			{
+				"name": "mk",
+				"secret": {
+					"secretName": "multikueue"
+				}
+			}
+		],
+		"containers": [
+			{
+				"name": "kubectl",
+				"image": "registry.k8s.io/kubectl:v1.28.4",
+				"env": [
+					{
+						"name": "KUBECONFIG",
+						"value": "/kubeconfig/worker1.kubeconfig"
+					}
+				],
+				"volumeMounts": [
+					{
+						"name": "mk",
+						"mountPath": "/kubeconfig"
+					}
+				], 
+				"args":[
+					"run", 
+					"testpod",
+					"--image='${E2E_TEST_IMAGE}'"
+				]
+			}
+		], 
+		"restartPolicy":"Never"
+	}
+}'
+kubectl run kubectl-to-2 --image registry.k8s.io/kubectl:v1.28.4 --overrides=\
+'
+{
+	"spec": {
+		"volumes": [
+			{
+				"name": "mk",
+				"secret": {
+					"secretName": "multikueue"
+				}
+			}
+		],
+		"containers": [
+			{
+				"name": "kubectl",
+				"image": "registry.k8s.io/kubectl:v1.28.4",
+				"env": [
+					{
+						"name": "KUBECONFIG",
+						"value": "/kubeconfig/worker2.kubeconfig"
+					}
+				],
+				"volumeMounts": [
+					{
+						"name": "mk",
+						"mountPath": "/kubeconfig"
+					}
+				], 
+				"args":[
+					"run", 
+					"testpod",
+					"--image='${E2E_TEST_IMAGE}'"
+				]
+			}
+		], 
+		"restartPolicy":"Never"
+	}
+}'
+kubectl wait pods kubectl-to-1 --for condition=Ready=False --timeout=180s
+kubectl wait pods kubectl-to-2 --for condition=Ready=False --timeout=180s
+
+sleep 10
```

Comment:
Please add a comment why the sleep is needed

Reply:
removed in favor of ginkgo based tests